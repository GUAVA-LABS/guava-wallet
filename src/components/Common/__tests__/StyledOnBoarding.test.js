import React from 'react';
import renderer from 'react-test-renderer';
import StyledOnBoarding from '../StyledOnBoarding';

test('Render StyledOnBoarding component', () => {
    const component = renderer.create(<StyledOnBoarding />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
