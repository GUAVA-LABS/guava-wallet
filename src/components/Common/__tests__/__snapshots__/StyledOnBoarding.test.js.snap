// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render StyledOnBoarding component 1`] = `
<div
  className="sc-bdVaJa iqYwhV"
/>
`;
